app/assets/v2/js/shared.js

```diff
@@ -1047,7 +1048,13 @@ function renderBountyRowsFromResults(results, renderForExplorer) {
       const timeLeft = timeDifference(dateNow, dateExpires);
       const expiredExpires = dateNow < dateExpires ? 'Expires' : 'Expired';
 
-      result['p'] += ('Opened ' + openedWhen + ' ago, ' + expiredExpires + ' ' + timeLeft);
+      if (isInfinite) {
+        const expiredExpires = 'Never expires';
```

Comment:
`expiredExpires` seems confusing as it's already declared above.
Would it make sense to change it to

```
if (isInfinite) {
  const expiredExpires = 'Never expires';
 ....
} else {
  const expiredExpires = dateNow < dateExpires ? 'Expires' : 'Expired';
  ...
}
```

Reply:
Totally agree, I had made it in that way not sure why I moved to outside the statement.